.github/workflows/test.yml

```diff
@@ -42,7 +42,7 @@ jobs:
 
       # Only install c-kzg if required. See https://github.com/ChainSafe/lodestar/pull/4888
       # Install in workspace root, else there are weird type errors with NodeJS types
-      - run: yarn add --ignore-workspace-root-check c-kzg
+      - run: yarn add --ignore-workspace-root-check c-kzg@1.0.7
```

Comment:
@g11tech is this install necessary still?

Reply:
not necessary on `unstable` since we do specify it in package.json https://github.com/ChainSafe/lodestar/blob/9c7c08b4adf9d04cd2e5f1084ed999a1a230a422/packages/beacon-node/package.json#L134

 but we do need that for `stable` since we only have `peerDependencies` there